### ISSUE TITLE
issue about loss.backward()

### ISSUE THREAD

**Issue by Jee-King** [Paper99/SRFBN_CVPR19#4]
Hi, thank you for sharing your work.
In the paper, I think that the calculation of lose is the sum of 4 iterations, and then backward.
In the code, lose is backward for each iteration.
I understand it right?

**Comment by Paper99** [comment]
They are equivalent, actually. 
The code is an effective way to implement multi-loss in our model.
You can find more details in this [question](https://discuss.pytorch.org/t/what-exactly-does-retain-variables-true-in-loss-backward-do/3508/12) and [answer](https://discuss.pytorch.org/t/what-exactly-does-retain-variables-true-in-loss-backward-do/3508/13).

**Comment by Jee-King** [comment]
Thank you for your reply.
